.changeset/many-toys-search.md

```diff
@@ -0,0 +1,24 @@
+---
+'sku': minor
+---
+
+Add `libraryFile` configuration option
+
+This option allows the file name of the library to be specified in the sku configuration.
+If this option is not specified then the `libraryName` option will be used for this purpose instead (note that this is the previously existing behaviour).
+
+**EXAMPLE USAGE**:
+
+```typescript
+// sku.config.ts
+import type { SkuConfig } from 'sku';
+
+const skuConfig: SkuConfig = {
+  libraryEntry: 'src/library.js',
+  renderEntry: 'src/render.js',
+  libraryName: 'MyAwesomeLibrary',
+  libraryFile: 'my-awesome-library',
+};
+
+export default skuConfig;
```

Comment:
Do we want to start encouraging people to use the [(new-ish) `satisfies` operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html)?

```suggestion
export default {
  libraryEntry: 'src/library.js',
  renderEntry: 'src/render.js',
  libraryName: 'MyAwesomeLibrary',
  libraryFile: 'my-awesome-library',
} satisfies SkuConfig;
```

Reply:
Agreed to look into this at a later date.